### ISSUE TITLE
Incorrect behaviour running Cypress from root with tests in subfolders

### ISSUE THREAD

**Issue by HugoGiraudel** [cypress-io/cypress#635]
### Setup

- Operating System: MacOS
- Cypress Version: 0.20.0
- Browser Version: n/a

### Is this a Feature or Bug?

A bug.

### Current behavior:

Considering the following structure:

```
_ package.json
_ node_modules
_ src/
  |_ clients/
    |_ foo/
      |_ cypress.json
      |_ cypress/
    |_ bar/
      |_ cypress.json
      |_ cypress/

```

And the following command run from root:

```
./node_modules/.bin/cypress run ./src/clients/foo
```

- A `cypress` folder gets created at root.
- The dummy tests are run instead of the expected ones.

### Desired behavior:

- No `cypress` folder gets created at root.
- The tests from `./src/clients/foo` are run.

### Additional infos

Running the following command doesn’t fix the problem (`--project`).

```
./node_modules/.bin/cypress run --project ./src/clients/foo
```

**Comment by brian-mann** [comment]
This should work with the `--project` flag. It uses `process.cwd()` by default unless that flag is set. The Cypress binary should then be told the correct project path.

We'll look into this.

**Comment by bahmutov** [comment]
Test project (should fail with 0.20.0) https://github.com/cypress-io/cypress-test-nested-projects

**Comment by brian-mann** [comment]
@bahmutov we are keeping issues open until they get released. Just tag as `pending release` or add a note if its really important. Gonna reopen until `0.20.1` comes out.

**Comment by brian-mann** [comment]
The code for this is merged, but it has not been released yet.

**Comment by brian-mann** [comment]
Fixed in [`0.20.1`](https://docs.cypress.io/guides/references/changelog.html#0-20-1).

**Comment by HugoGiraudel** [comment]
Yay, thank you!
